plugin/pkg/shoot/validator/admission.go

```diff
@@ -40,6 +40,7 @@ import (
 	"k8s.io/apiserver/pkg/authorization/authorizer"
 	kubeinformers "k8s.io/client-go/informers"
 	kubecorev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/utils/pointer"
```

Comment:
Use `ptr` package only?

Reply:
Ahh right, forgot for a second that `pointer` functions are deprecated and we removed them. Let me see if I ca adapt this.

Reply:
Done